Sources/PopupBridge/POPPopupBridgeDelegate.swift

```diff
@@ -2,16 +2,6 @@ import UIKit
 
 /// Set a delegate to handle various events the "popup" lifecycle.
 @objc public protocol POPPopupBridgeDelegate: NSObjectProtocol {
-    
-    /// Popup Bridge will provide a Safari View Controller to your delegate.
-    /// You must present the view controller in this method.
-    @objc func popupBridge(_ bridge: POPPopupBridge, requestsPresentationOfViewController viewController: UIViewController)
-
-    /// You must dismiss the view controller in this method.
-    @objc func popupBridge(_ bridge: POPPopupBridge, requestsDismissalOfViewController viewController: UIViewController)
-    
-    /// Optional: Receive the URL that will be opened
-    @objc optional func popupBridge(_ bridge: POPPopupBridge, willOpenURL url: URL)
 
     /// Optional: Receive messages from the web view
     @objc optional func popupBridge(_ bridge: POPPopupBridge, receivedMessage messageName: String, data: String?)
```

Comment:
I don't think we need `POPPopupBridgeDelegate` at all anymore. The optional `receivedMessage()` function was really just a nice-to-have by product of our SDKs communication with `WKScriptMessageHandler` that we decided to surface to the merchant in case they wanted it. Since it isn't necessary for a PopupBridge integration, I think we should remove it.

Reply:
Done: https://github.com/braintree/popup-bridge-ios/pull/57/commits/93c4c3662227d1b5519859b0555507cd59f81505 always down to 🔥